reverseVowels.js

```diff
@@ -0,0 +1,6 @@
+// In this kata, your goal is to write a function which will reverse the vowels in a string. Any characters which are not vowels should remain in their original position.
+function reverseVowels(str) {
+    const vowels = str.match(/[aeiou]/gi);
+    const result = str.replace(/[aeiou]/gi, () => vowels.pop());
+    return result
```

Comment:
Creating a variable only to just return it on the next line is quite redundant, I would recommend trying this 
```suggestion
return  str.replace(/[aeiou]/gi, () => vowels.pop());

```